mobify/mobify-code-style
# Renaming files to avoid these files being flagged as invalid js files.

Renaming jshintrc.js -> .jshintrc and jscsrc.js to -> .jscsrc. As JavaScript files these files were being flagged as invalid; it's not valid js to define an object that's not part of an assignment expression. 

Status: **Ready to merge**

Reviewers: @MikeKlemarewski @stewartyu 
## Changes
- renamed files
## How to test-drive this PR
- checkout this branch
- ensure files are renamed to .jshintrc and .jscsrc respectively
